EasyClangComplete.py

```diff
@@ -169,7 +169,22 @@ def on_close(self, view):
             log.debug(" closing view %s", view.buffer_id())
             if not self.completer:
                 return
-            self.completer.remove(view.buffer_id())
+            future = EasyClangComplete.pool_read.submit(
+                self.completer.remove, view.buffer_id())
+            future.add_done_callback(EasyClangComplete.completer_removed)
```

Comment:
Yes, exactly what I was thinking, but this would either block or make things even more complicated. Probably, the proper way would be to have a thread that waits will there are no more jobs present and then would remove the TU. But then there has to be a block that you cannot submit new jobs while this one waits and so on...